packages/react-router/docs/api/withRouter.md

```diff
@@ -27,3 +27,14 @@ class ShowTheLocation extends React.Component {
 // terminology) to the router.
 const ShowTheLocationWithRouter = withRouter(ShowTheLocation)
 ```
+
+**IMPORTANT NOTE:** If you are using withRouter to prevent shouldComponentUpdate blocks, it is important that withRouter wraps the component that implements shouldComponentUpate. For example, when using Redux: 
+
+```js
+// This gets around shouldComponentUpdate
+withRouter(connect(...)(MyComponent)
+// This does not
+connect(...)(withRouter(MyComponent))
+```
+
+See [this post](https://github.com/ReactTraining/react-router/blob/v4.0.0-beta.8/packages/react-router/docs/guides/blocked-updates.md) for more information.
```

Comment:
This should link to master instead of the beta tag. Sorry, things are moving around at the moment!